spec/export/index.md

```diff
@@ -96,21 +96,38 @@ Export data from a FHIR server, whether or not it is associated with a patient.
 
 #### Query Parameters
 
-- ```_outputFormat``` (string, optional, defaults to ```application/fhir+ndjson```)
-
-  The format for the requested bulk data files to be generated as per [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html). Defaults to `application/fhir+ndjson`. Servers SHALL support [Newline Delimited JSON](http://ndjson.org), but MAY choose to support additional output formats. Servers SHALL accept the full content type of ```application/fhir+ndjson``` as well as the abbreviated representations ```application/ndjson``` and ```ndjson```.
-
-- ```_since``` (FHIR instant type, optional)  
-
-  Resources will be included in the response if their state has changed after the supplied time (e.g.  if Resource.meta.lastUpdated is later than the supplied `_since time`).
-
-- ```_type``` (string of comma-delimited FHIR resource types, optional)
-
-  Only resources of the specified resource types(s) SHALL be included in the response. If this parameter is omitted, the server SHALL return all supported resources within the scope of the client authorization. For Patient- and Group-level requests, the [Patient Compartment](https://www.hl7.org/fhir/compartmentdefinition-patient.html) SHOULD be used as a point of reference for recommended resources to be returned. However, other resources outside of the patient compartment that are helpful in interpreting the patient data (such as Organization and Practitioner) may also be returned. Servers unable to support `_type` SHOULD return an error and OperationOutcome resource so clients can re-submit a request omitting the `_type` parameter.
+<table class="table">
+  <thead>
+    <th>Parameters</th>
+    <th>Optionality</th>
+    <th>Type</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>```_outputFormat```</td>
+      <td>Optional</td>
+      <td>String</td>
+      <td>  The format for the requested bulk data files to be generated as per [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html). Defaults to `application/fhir+ndjson`. Servers SHALL support [Newline Delimited JSON](http://ndjson.org), but MAY choose to support additional output formats. Servers SHALL accept the full content type of ```application/fhir+ndjson``` as well as the abbreviated representations ```application/ndjson``` and ```ndjson```.</td>
```

Comment:
@ricky1uphealth The `-based formatting doesn't work inside HTML tables (or at least, isn't rendering for me). I think we need `<code>` formatting for these.

Reply:
got it, we have to use code tags for it work well, fixed here https://github.com/HL7/bulk-data/pull/34/commits/b090adf1b0a7427d0ddcb7eda99275d238142859